app/src/main/java/edu/wpi/first/shuffleboard/app/PreloaderController.java

```diff
@@ -4,16 +4,26 @@
 
 import org.controlsfx.tools.Utils;
 
+import java.util.Random;
+
 import javafx.fxml.FXML;
 import javafx.scene.control.Label;
 import javafx.scene.control.ProgressBar;
 import javafx.scene.layout.Pane;
+import javafx.scene.paint.Color;
 
 public class PreloaderController {
 
+  private static final Color primaryColor = Color.rgb(39, 39, 247);
+  private static final Color secondaryColor = Color.rgb(64, 64, 255);
```

Comment:
How do you feel about using colors that complement our branding guide?

https://github.com/wpilibsuite/branding

Reply:
This color scheme matches the default theme.  It'd be a bit weird to have a red-and-gray preloader switch to a blue-and-white theme when shuffleboard is launched.

If the WPILib logo is added, I would switch to a red color scheme.  I don't know where the logo is actually used besides the developer shirts; it's definitely not in any other desktop tool, so it's not something that people would recognize and immediately recognize as "WPILib"

Reply:
The installer will have it, and vscode as the extension icon, but it won't be anywhere actually in the extension